boot/makebootable.go

```diff
@@ -330,6 +330,18 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		return fmt.Errorf("cannot set run system environment: %v", err)
 	}
 
+	_, ok = bl.(bootloader.ManagedAssetsBootloader)
```

Comment:
the first UC20 TODO at the beginning of this function I think can be removed now?

`- create grub.cfg instead of using the gadget one`

Reply:
Dropped